src/i18n/locales/fr.json

```diff
@@ -0,0 +1,83 @@
+  "blog.authors.list.title.by": "par",
+  "blog.authors.list.title.and": "et",
+  "components.downloadAdditional.title": "Téléchargements supplémentaires",
+  "components.downloadHeader.navigation.activeSection": "Téléchargements",
+  "components.downloadHeader.navigation.title": "Téléchargements",
+  "components.downloadHeader.navigation.nodeVersion": "{lts, sélectionnez, vrai {LATEST LTS}other {CURRENT}} VERSION {nodeVersion}",
+  "components.downloadHeader.navigation.npmVersion": "(comprend npm {npmVersion})",
+  "components.downloadTable.header.release": "Publication",
+  "components.downloadTable.header.status": "Statut",
+  "components.downloadTable.header.codename": "Nom de code",
+  "components.downloadTable.header.initialRelease": "Version initiale",
+  "components.downloadTable.header.ltsStarts": "Active LTS Start",
+  "components.downloadTable.header.maintenanceStart": "Maintenance LTS Start",
+  "components.downloadTable.header.endOfLife": "Fin de vie",
+  "components.downloadReleases.upcomingReleases.title": "Communiqués de presse à venir",
+  "components.downloadReleases.upcomingReleases.content": "Les versions majeures de Node.js entrent dans le statut Current release pendant six mois, ce qui donne aux auteurs de bibliothèques le temps d'ajouter la prise en charge de ces versions. Après six mois, les versions impaires (9, 11, etc.) ne sont plus prises en charge et les versions paires (10, 12, etc.) passent au statut Active LTS et sont prêtes pour une utilisation générale. Le statut de la version LTS est un \"support à long terme\", qui garantit généralement que les bogues critiques seront corrigés pendant un total de 30 mois. Les applications de production ne doivent utiliser que les versions Active LTS ou Maintenance LTS.",
+  "components.downloadToggle.current": "Actuel",
+  "components.downloadToggle.lts": "LTS",
+  "components.downloadToggle.recommendation": "{selected, select, LTS {Recommended for most users}other {With the latest features}}",
+  "components.editLink.title": "Modifier cette page sur GitHub",
+  "components.footer.links.trademark": "Politique en matière de marques",
+  "components.footer.links.privacy": "Politique de confidentialité",
+  "components.footer.links.codeOfConduct": "Code de conduite",
+  "components.footer.links.security": "Rapports de sécurité",
+  "components.footer.links.about": "À propos",
+  "components.footer.links.blog": "Blog",
+  "components.footer.links.openJsFoundation": "©Fondation OpenJS",
+  "components.footer.links.github": "GitHub",
+  "components.header.links.learn": "Apprendre",
+  "components.header.links.docs": "Docs API",
+  "components.header.links.download": "Télécharger",
+  "components.header.links.community": "Communauté",
+  "components.hero.downloadLts": "Télécharger Node(LTS)",
+  "components.hero.currentVersion": "{isLts, select, true {Version {currentVersion} - }other {}}",
```

Comment:
We changed this line in the latest version of the `en.json`

Reply:
OK i chez that

Reply:
I done